modules/vba_documents/app/sidekiq/vba_documents/upload_processor.rb

```diff
@@ -90,7 +90,8 @@ def download_and_process
         brt = Benchmark.realtime do
           # Validations
           validate_parts(@upload, parts)
-          validate_metadata(parts[META_PART_NAME], submission_version: @upload.metadata['version'].to_i)
+          validate_metadata(parts[META_PART_NAME], @upload.consumer_id, @upload.guid,
+                            submission_version: @upload.metadata['version'].to_i)
```

Comment:
pass in the consumer_id to check to see if they have any allowed custom metadata, upload guid is for logging